airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -136,13 +136,17 @@ class ECSOperator(BaseOperator):
         Only required if you want logs to be shown in the Airflow UI after your job has
         finished.
     :type awslogs_stream_prefix: str
+    :param quota_retry: Config if and how to retry _start_task() for transient errors.
+    :type quota_retry: dict
```

Comment:
`_start_task()` is a private function, and an end user wouldn’t have a clue what this means without reading the source code. So this is a good change to rewrite the doc to describe what `_start_task()` actually does and how this config interacts with it.